front/src/main.js

```diff
@@ -3,6 +3,7 @@ import App from './App.vue';
 import router from '@router';
 import store from '@store';
 import vuetify from './plugins/vuetify';
+import axios from 'axios';
 
 import Default from './layouts/Default.vue';
```

Comment:
please, add @layout path and use it like

import { Default, Authorize } from '@layout'